Makefile.PL

```diff
@@ -0,0 +1,52 @@
+use 5.014;
+use strict;
+use warnings;
+use ExtUtils::MakeMaker;
+
+WriteMakefile(
+    NAME               => 'Locale::Maketext::Test',
+    AUTHOR             => q{Binary.com <binary@cpan.org>},
+    VERSION_FROM       => 'lib/Locale/Maketext/Test.pm',
+    ABSTRACT           => 'Locale::Maketext::Test supports testing of po files',
```

Comment:
The name isn't needed here, since the abstract is usually displayed [next to the name](https://metacpan.org/author/BINARY) anyway - maybe just "test validity of .po translation files" would do?